### PR TITLE
Created a Speed checking script in python 3 named TestMyInternetSpeed

### PR DESCRIPTION
**Pull request by thesmartdeveloperr**
hey, i have written a small script for checking the net speeds using python, my script can also check for pings.

### CODE REVIEW

**Comment by thesmartdeveloperr** [comment]
@hastagAB can you review this please?

File: README.md [hunk 1]
```diff
@@ -1,233 +0,0 @@
-﻿# Awesome Python Scripts :sunglasses: <img alt="PyPI" src="https://warehouse-camo.cmh1.psfhosted.org/18509a25dde64f893bd96f21682bd6211c3d4e80/68747470733a2f2f696d672e736869656c64732e696f2f707970692f707976657273696f6e732f64796e61636f6e662e737667"> [![HitCount](http://hits.dwyl.io/hastagAB/Awesome-Python-Scripts.svg)](http://hits.dwyl.io/hastagAB/Awesome-Python-Scripts)  ![GitHub stars](https://img.shields.io/github/stars/hastagAB/Awesome-Python-Scripts?style=social)
```
Comment (hastagAB) [inline README.md:1]: Did you accidentally delete this file?
This is perhaps one of the most important files in any open source project, please revert this commit and add this file.

**Review by hastagAB — COMMENTED** [review]
Thanks, @thesmartdeveloperr 
The Script LGTM! 👍🏻 
If possible, do try to squash your commits.

**Comment by thesmartdeveloperr** [comment]
@hastagAB  **pls review the changes**, i have _updated the readme file i removed by mistake_, and am _not sure how to squash the commits_ as i deleted my local copy earlier and now i am not sure how to do that.
Do let me know if i need to change something or how to squash, if its necessary.

**Review by hastagAB — APPROVED** [review]

**Pull request merged**